pageserver/src/branches.rs

```diff
@@ -98,8 +98,11 @@ pub fn init_repo(conf: &'static PageServerConf, repo_dir: &Path) -> Result<()> {
     // and we failed to run initdb again in the same directory. This has been solved for the
     // rapid init+start case now, but the general race condition remains if you restart the the
     // server quickly.
-    let repo = crate::repository::rocksdb::RocksRepository::new(
+    let storage = crate::sled_storage::SledObjectStore::create(conf)?;
```

Comment:
Perhaps. I'm thinking that this is a bit of a wart in the abstraction, as we've hardcoded the kind of object store here. Of course, we have to specify it somewhere, and I think this is fine for now. But this module shouldn't depend on the rocksdb or sled_storage module otherwise, hence no `use`. I'm not sure if that distinction makes sense to anyone but me :-).

Reply:
>  But this module shouldn't depend on the rocksdb or sled_storage module otherwise, hence no use. I'm not sure if that distinction makes sense to anyone but me :-).

Yeah, that's not really a factor in Rust. There's no reason to care which modules invoke code from other modules other than readability.